torch/onnx/symbolic_opset9.py

```diff
@@ -7,8 +7,8 @@
 import functools
 import math
 import sys
+import typing
 import warnings
-from typing import List, Optional, Tuple, Union
```

Comment:
Was there a lint error caused by this? Let's see how we can keep importing from typing and keep lint happy

Reply:
Yes...

Reply:
Do we define `__all__` in symbolic_opset12? If not we need it

Reply:
Not sure why it is interfering with this pr though… this pr doesn’t touch opset12 right

Reply:
`opset12` doesn't have `__all__`, so I will add it. Error message is copied from another PR.
For this PR, I will add `typing.*` to `opset9`'s `__all__`.

Reply:
typing does not belong to all. All should only contain public apis we want to expose from torch.onnx. Once we have all with all the public functions, the test will know typing is not part of the public interface

Reply:
For this pr in particular, I don’t expect to see the error you pasted. Feel free to just import from typing, I can take a look at the tests